lib/tasks/web.rake

```diff
@@ -74,6 +74,33 @@ namespace :slack do
           'bin/commands.rb',
           commands_template.result(files: data.keys.map { |key| key.tr('.', '_') })
         )
+
+        ### Errors
```

Comment:
I started out with all of this new stuff in its own rake task, `web_errors.rake`, but then after living with that for a bit I thought it was wrong to separate. Again I could go either way here.

Reply:
I'd split it, but NBD.